models/logger.go

```diff
@@ -71,3 +71,32 @@ func (l *Logger) Delete() (err error) {
 	}
 	return nil
 }
+
+// LogMessage holds the message payload
+type LogMessage struct {
+	Message   string `json:"message"`
+	Timestamp string `json:"timestamp"`
```

Comment:
I think the timestamp can be set at the other side of nats, by logger so we don't really need it here.